BMM.Core/ViewModels/Parameters/Interface/ITrackCollectionParameter.cs

```diff
@@ -0,0 +1,9 @@
+namespace BMM.Core.ViewModels.Parameters.Interface
+{
+    public interface ITrackCollectionParameter
+    {
+        int? TrackCollectionId { get; }
+        string Name { get; }
+        string SharingSecret { get; }
```

Comment:
I think it's better to have the SharingSecret separately, e.g. in a SharingSecretParameter